vinnie/base.py

```diff
@@ -84,7 +83,7 @@ def push(self, remote):
         if self.config.push:
             self.backend.push(remote)
         else:
-            warnings.warn("Skipping push.", UserWarning)
+            print("Skipping push.", file=sys.stderr)
```

Comment:
shouldn't we use `click.echo` instead of `print`?

Reply:
Note that `sys.stderr` is not the same as `sys.stdout` which is where I'd expect `click.echo` to go.

I don't think you had `click` available in this code. I'm happy to use it otherwise. I don't have a strong opinion about implementation details so long as it doesn't behave as it does today.

Reply:
Let's use `click.echo("Skipping push.", err=True)`